cmd/status-backend/main.go

```diff
@@ -57,3 +61,17 @@ func main() {
 	srv.RegisterMobileAPI()
 	srv.Serve()
 }
+
+// haltOnInterruptSignal catches interrupt signal (SIGINT) and
+// stops the node. It times out after 5 seconds
```

Comment:
Where is this 5 seconds timeout defined? :thinking:

Reply:
Timeout was a good thing, no?

Reply:
@osmaczko I think the timeout should be defined by externally. One could always force quit the process without expecting it for graceful shutdown. And I'm not sure what would be the timeout, how would we define it 🤔 

And anyway, the comment was misliading, there's no timeout for this function `statusd`

Reply:
Yeah, I agree, good point. It should be up to tests to send SIGKILL on timeout.